src/main/java/xyz/holocons/mc/holoitemsrevamp/enchant/EnchantListener.java

```diff
@@ -99,6 +117,32 @@ public void onProjectileLaunch(ProjectileLaunchEvent event) {
         });
     }
 
+    /**
+     * Handles PlayerDeath enchantments.
+     *
+     * @param event The PlayerDeathEvent
+     */
+    @EventHandler(ignoreCancelled = true)
+    public void onPlayerDeath(PlayerDeathEvent event) {
+
+        final Player player = event.getPlayer();
+        final PlayerInventory inventory = player.getInventory();
+        for (ItemStack item : inventory) {
+
+            if (item == null){
+                return;
```

Comment:
Why is there a return here?

Reply:
Ah. Was under the assumption that it returned null for every slot with no items.

 Now that I think about it, it would not matter either way.